eth/tracers/native/call_parity.go

```diff
@@ -104,10 +105,10 @@ func (t *callParityTracer) CaptureTxEnd(restGas uint64) {}
 
 // NewCallParityTracer returns a native go tracer which tracks
 // call frames of a tx, and implements vm.EVMLogger.
-func NewCallParityTracer(ctx *tracers.Context) tracers.Tracer {
+func NewCallParityTracer(ctx *tracers.Context, j json.RawMessage) (tracers.Tracer, error) {
```

Comment:
@ziogaschr This type (an interface type, I think) changes throughout the tracer implementations. I added these signature values to the Parity and StateDiff tracers. PTAL if what I did here is OK.